### ISSUE TITLE
Optimizing for wait time

### ISSUE THREAD

**Issue by loisaidasam** [erikbern/mta#2]
Hello, excellent post. No comment section, but I had a thought.

In your post you write:

> Let’s assume you want to optimize for a wait time that’s less than 30 min in 90% of the cases. Then the max time you should wait is about 11 minutes until giving up (this is at the point where the yellow line cuts the 30 min mark).

But if you want to optimize for your TOTAL wait time being less than 30 min in 90% of the cases, isn't that the point at which `time you have waited` + `additional wait time until subway arrives` <= `30`?

Specifically, isn't the answer actually somewhere around the 8 minute mark?

![time_to_arrival_percentiles_annotated](https://cloud.githubusercontent.com/assets/213281/14368099/a360ae2a-fce9-11e5-860e-9f4c3955af41.png)

Maybe my interpretation is wrong, food for thought.

**Comment by erikbern** [comment]
Yeah that probably makes sense. But you don't know that until after 8 minutes. And at that point, the 8 minutes are "sunk cost" anyway :)
